scripts/webpack/replaceImagesWithRequires.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ Custom Webpack loader. Takes a string referring to a filename,
+ runs it through gray-matter (https://www.npmjs.com/package/gray-matter)
+ to split the front matter and content out, then returns just the content.
+ */
```

Comment:
I copypasta'd this, needs changing

```suggestion
/*
 Custom Webpack loader. 
 Replaces relative image references in Markdown files with React components using a
 `require` statement, so that later in the process they can be captured by `file-loader`.
 
 Turns:
 ![alt text](./image.png)
 
 into:
 <img src={'/' + require('./image.png'} alt="alt text" />
 */
```

Reply:
Lol this is your code 😛